pow/web3j/src/test/resources/geth/README.md

```diff
@@ -0,0 +1,14 @@
+Go-Ethereum instance with open RPC and non-persistent DB initialized with `genesis.json`.   
+
+Building image:
+```shell script
+docker build .
+```
+Running container:
+```shell script
+docker run -d -p 8545:8545 -p 30303:30303 <image id>
```

Comment:
It would be better to give this image a name: `docker build -t name && docker run -d name`.